### ISSUE TITLE
No one can log in to the website. Tried making a new admin and still cannot login. 

### ISSUE THREAD

**Issue by thezalkor** [grokability/snipe-it#13208]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

First time setting up snipe-it. Fresh install on Ubuntu vm instance. It was working fine but I went to add a certificate to make it more secure (https). This was done through certbot. I thought I made the proper changes in the virtual host file and for my domain a record. I can get to the site just fine, the login screen but when I type my username and password it reloads and does nothing after that. 

Nothing shows up in the logs or using debug mode either. Using Chrome developer tools to see if anything pops up and only extra code that pops up is a 302 code. 

I have checked the virtual host file again and have searched online and in this forum to see if there is anything similar but I can't seem to find anything that helps. So I am stuck. 

### Reproduction steps

1.  Type username and password
2. Press Login
3. Reloads page and starts over
...

### Expected behavior

Being able to login. 

### Snipe-IT Version

pulled from github two weeks ago. 

### Operating System

Ubuntu 20.04

### Web Server

Apache 2.4.41

### PHP Version

7.4.3

### Device

MacBook Pro

### Operating System

MacOS Ventura 13.4

### Browser

Chrome

### Version

 114.0.5735.133

### Additional context

This is a fresh install. 
Running through a VM instance of Ubuntu on Google Cloud Console. 
Manual Installation
This was working with a few 500 error codes popping up when trying to upload files but at least we were able to create assets and use the system
I have added a new admin user trying to see if I could sign in with a new user ( did not work same issue)

**Comment by marcusmoore** [comment]
This is interesting....is there anything popping up in apache's logs?

**Comment by thezalkor** [comment]
So from what I can tell it seems to be redirecting me back to the login page. Not 100% sure why. I ended up just starting over. This time I paid more attention to what had what permissions and made sure things were in the correct spots. My new site and VM instance are working beautifully. 

So I couldn't fully figure it out but since it was so early in the setup (meaning I had just finished setting it up and hadn't fully uploaded any assets or started to configure the site) I just hit the reset button lol. 

Thanks for the reply but I will close the ticket now. Unless you want more info?
